apps/frontend/src/libs/hooks/use-popover-position/use-popover-position.hook.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+type Properties = {
+	top?: number;
+};
+
+const usePopoverPosition = <T extends HTMLElement>(
+	reference: React.RefObject<T>,
+	isOpen: boolean,
+	hasPositioning: boolean = false,
+): Properties => {
+	const [position, setPosition] = useState<Properties>({});
+
+	useEffect(() => {
+		if (!hasPositioning || !isOpen || !reference.current) {
+			setPosition({});
+
+			return;
+		}
+
+		const { top } = reference.current.getBoundingClientRect();
+		const scrollOffset = window.scrollY || document.documentElement.scrollTop;
```

Comment:
I meant to use either `window.scrollY` or `document.documentElement.scrollTop` only in my first question

Reply:
I prefer to leave both, but I can choose one (window.scrollY modern option)

Reply:
I think we need to leave document.documentElement.scrollTop only, it should cover all the cases

Reply:
noted